packages/client/hmi-client/src/workflow/ops/simulate-julia/tera-simulate-node-julia.vue

```diff
@@ -159,21 +159,24 @@ const watchCompletedRunList = async (runIdList: string[]) => {
 	);
 	runResults.value = newRunResults;
 
+	const port = props.node.inputs[0];
+
 	const state = _.cloneDeep(props.node.state);
 	if (state.simConfigs.chartConfigs.length === 0) {
 		state.simConfigs.chartConfigs.push([]);
 	}
 	state.simConfigs.runConfigs[runIdList[0]] = {
 		runId: runIdList[0],
-		active: true
+		active: true,
+		configName: port.label,
+		timeSpan: state.currentTimespan
```

Comment:
You changed timeSpan from string => Timespan, so when you are making assignment it is by reference and not by value. Can you confirm if this is okay? Or if we need to clone a copy of it?

Ditto for the CIEMSS side of things.

Reply:
Just did some testing with updating the timespan in the UI and it seems like the initial assignment is preserved. But should I clone it just to be extra safe?

Reply:
Updated to assign a copy of `state.currentTimespan` instead of direct assignment.